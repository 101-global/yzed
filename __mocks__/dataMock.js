export const model = {
  name: 'YZED',
  brandName: 'YZED',
  imageSlug: 'yzed',
  glbFile: 'https://www.testYZEDfile.com/glbFile.glb',
  usdzFile: 'https://www.testYZEDfile.com/usdzFile.usdz',
  animatedGlbFile: 'https://www.testYZEDfile.com/animatedGlbFile.glb',
  exposure: 1,
};
