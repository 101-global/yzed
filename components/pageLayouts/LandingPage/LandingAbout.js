import React, { useContext, useEffect, useState } from 'react';

import BlackLink from '../../reusableStyledComponents/BlackLink';
import ProgressiveImage from '../../ProgressiveImage';
import YZEDAnimatedLogo from '../../reusableStyledComponents/YZEDAnimatedSVG';
import selfExpressionWhite from '../../../public/self-expression.svg';
import theme from '../../../utilities/theme';

const LandingAbout = ({ model }) => {
  const imageBaseUrl = 'https://oneoone-resource.s3.ap-northeast-2.amazonaws.com/yzed/';
  const smallPhoto = {
    id: `${model.name} Detail View 2`,
    mini: `${imageBaseUrl}${model.imageSlug}_image_2_mini-compressor.jpg`,
    thumb: `${imageBaseUrl}${model.imageSlug}_image_2_thumb-compressor.jpg`,
    max: `${imageBaseUrl}${model.imageSlug}_image_2_max-compressor.jpg`,
  };
  const largePhoto = {
    id: `${model.name} Detail View 3`,
    mini: `${imageBaseUrl}${model.imageSlug}_image_3_mini-compressor.jpg`,
    thumb: `${imageBaseUrl}${model.imageSlug}_image_3_thumb-compressor.jpg`,
    max: `${imageBaseUrl}${model.imageSlug}_image_3_max-compressor.jpg`,
  };

  return (
    <>
      <div className='main-about-container pb-16 lg:pb-32 align-center justify-center w-full'>
        <div className='text-container grid grid-cols-1 lg:block align-center justify-center w-full lg:w-4/5 mx-auto my-0 relative'>
          <h1 className='text-3xl lg:text-4xl px-4 lg:pb-8 lg:p-0 lg:text-4xl w-full lg:w-5/12 inline-block text-center lg:text-left'>
            “The desire to make fashion more accessible to people.”
          </h1>
          <p className='text-xl w-full lg:w-6/12 block lg:inline-block relative lg:absolute right-o font-hairline z-50 right-0 px-4 py-12 lg:pb-8 lg:pl-8 lg:pt-0 lg:pr-0 right-0 text-justify'>
            We are an international team of four, based in Seoul, South Korea. We became united by
            the desire to make fashion more accessible to people, which has continued to be our
            team’s top priority. We strongly believe that a change of status quo is imminent in the
            fashion industry - and that it’s going to be catal<strong>yzed</strong> by none other
            than digital garments created in 3D & AR. We strive to create a community where our
            users self-expression leads our decisions.
          </p>
        </div>
        <div className='bottom-section relative'>
          <div className='image-container w-full'>
            <div className='small-image-container relative lg:absolute right-0  lg:w-2/5 mb-8 lg:mb-0'>
              <ProgressiveImage
                image={largePhoto.max}
                preview={largePhoto.mini}
                alt={largePhoto.id}
              />
            </div>
            <div className='large-image-container  lg:w-9/12'>
              <ProgressiveImage
                image={smallPhoto.max}
                preview={smallPhoto.mini}
                alt={smallPhoto.id}
              />
            </div>
            <div className='word-mark'>
              <img src={selfExpressionWhite} alt='contact us' />
            </div>
          </div>
        </div>
      </div>

      <style jsx>{`
        .main-about-container {
          color: white;
        }
        h1,
        p {
          color: ${theme.colors.white};
          background: ${theme.colors.black};
        }
        .text-container {
          padding-top: 100px;
        }
        h1 {
          font-weight: 600;
        }
        p {
          font-weight: 100;
        }
        .small-image-container {
          top: calc(50% - 140px);
          z-index: 100;
        }
        .word-mark img {
          overflow: hidden;
          position: absolute;
          bottom: -30px;
          right: 80px;
          mix-blend-mode: difference;
          filter: grayscale(100%) !important;
        }
        @media (max-width: 768px) {
          .word-mark img {
            max-width: 100%;
            right: 0;
            left: 0;
            bottom: -20px;
          }
        }
      `}</style>
    </>
  );
};

export default LandingAbout;
