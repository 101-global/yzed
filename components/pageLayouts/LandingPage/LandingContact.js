import React, { useContext, useEffect, useState } from 'react';

import ContactAnimated from '../../reusableStyledComponents/ContactAnimatedSVG';
import theme from '../../../utilities/theme';

const LandingContact = () => {
  return (
    <>
      <>
        <div className='main-contact-container px-4 lg:px-dsk flex flex-col lg:flex-row justify-center lg:justify-between align-center w-full'>
          <div className='title-section flex flex-col justify-center align-center w-full lg:w-6/12 relative'>
            <h1 className='text-3xl lg:text-5xl m-0 lg:mr-16 text-center lg:text-right mt-20 mb-20 lg:my-0 font-semibold z-fiveHund'>
              Let's Work Together!
            </h1>
            <ContactAnimated />
          </div>
          <div className='content-section flex flex-col w-full lg:w-6/12 md:mb-16 lg:mb-0 text-justify lg:text-left justify-center align-center relative'>
            <p className='text-xl font-light'>
              Dive into a mixed-reality world with a team of professionals in 3D & graphic design,
              UI/UX, trend forecasting, as well as front and back-end programming. YZED is turning
              your exquisite physical products into digital-dazzle, leading your brand into the
              digital era.
            </p>
            <div className='signature mt-16 text-left'>
              <p className='text-xl font-light'>Interested in working with us? </p>
              <p className='text-xl font-light'>
                Please contact us at{' '}
                <a className='email-link font-semibold inline-block cursor-pointer'>
                  yzed@101.global
                </a>
              </p>
            </div>
          </div>
        </div>
      </>
      <style jsx>{`
        a.email-link:after {
          content: '';
          background-image: linear-gradient(to right, #a891d9, #73cfdb, #a891d9, #73cfdb);
          background-size: 260% 200%;
          width: 110%;
          margin-left: -5%;
          margin-top: -10px;
          height: 12px;
          display: block;
        }
        a.email-link:hover:after {
          animation: gradient 3s ease infinite;
          animation-direction: alternate;
        }
        @keyframes gradient {
          0% {
            background-position: 0% 50%;
          }
          50% {
            background-position: 100% 50%;
          }
          100% {
            background-position: 0% 50%;
          }
        }
      `}</style>
    </>
  );
};

export default LandingContact;
