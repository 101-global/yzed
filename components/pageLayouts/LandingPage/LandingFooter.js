import React, { useContext, useEffect, useState } from 'react';

import theme from '../../../utilities/theme';

const LandingFooter = () => {
  return (
    <>
      <footer className='text-left'>
        <p className='p-2 lg:px-dsk text-md lg:text-md'>
          <a className='color-link' href='/'>
            YZED
          </a>{' '}
          - All rights reserved to{' '}
          <a href='https://www.101.global/?lang=en' target='_blank' rel='noopener noreferrer'>
            101Global
          </a>{' '}
          and{' '}
          <a href='https://www.hanwha.com/en.html' target='_blank' rel='noopener noreferrer'>
            Hanwha
          </a>
          .
        </p>
      </footer>
      <style jsx>{`
        footer p {
          font-weight: 200;

          background: ${theme.colors.black};
          color: ${theme.colors.white};
          width: 100%;
          letter-spacing: 0.02em;
        }

        footer p a {
          font-weight: 600;
          display: inline-block;
        }
        .color-link:after {
          content: '';
          background-image: linear-gradient(to right, #a891d9, #73cfdb, #a891d9, #73cfdb);
          background-size: 260% 200%;
          width: 110%;
          margin-left: -5%;
          margin-top: -12px;
          height: 12px;
          display: block;
        }
        .color-link:hover:after {
          animation: gradient 3s ease infinite;
          animation-direction: alternate;
        }
        @keyframes gradient {
          0% {
            background-position: 0% 50%;
          }
          50% {
            background-position: 100% 50%;
          }
          100% {
            background-position: 0% 50%;
          }
        }
      `}</style>
    </>
  );
};

export default LandingFooter;
