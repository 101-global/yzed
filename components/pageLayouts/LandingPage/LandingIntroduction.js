import React, { useContext, useEffect, useState } from 'react';

import BlackLink from '../../reusableStyledComponents/BlackLink';
import ProgressiveImage from '../../ProgressiveImage';
import PropTypes from 'prop-types';
import YZEDAnimatedLogo from '../../reusableStyledComponents/YZEDAnimatedSVG';
import theme from '../../../utilities/theme';

const LandingIntroduction = ({ model }) => {
  const imageBaseUrl = 'https://oneoone-resource.s3.ap-northeast-2.amazonaws.com/yzed/';
  const photo = {
    id: `${model.name} Detail View 1`,
    mini: `${imageBaseUrl}${model.imageSlug}_image_1_mini-compressor.jpg`,
    thumb: `${imageBaseUrl}${model.imageSlug}_image_1_thumb-compressor.jpg`,
    max: `${imageBaseUrl}${model.imageSlug}_image_1_max-compressor.jpg`,
  };

  return (
    <>
      <div className='main-intro-container p-0 lg:pl-dsk grid gap-8 lg:gap-16 sm:grid-cols-1 lg:grid-cols-2 mb-16 lg:mb-0'>
        <div className='flex w-full mt-16 md:mt-24 lg:mt-0 text-center lg:text-left relative'>
          <div className='flex flex-col justify-between font-black'>
            <YZEDAnimatedLogo />
            <h1 className='pt-16 px-4 lg:px-0 text-4xl lg:text-5xl my-0 mx-auto text-center lg:text-left font-semibold'>
              Where Democratic Fashion catalyzes self-discovery.
            </h1>
            <p className='para-standard text-justify lg:text-left px-4 lg:px-0 py-12 lg:py-0 my-0 mx-auto lg:mt-24 lg:m-0'>
              Democratic Fashion is a practice that minimizes the gap between a designersâ€™ work and
              the mass market, while considering fair economic and social trade off. YZED is built
              upon the five most important factors of this practice, which you can read more about
              in our manifesto.
            </p>
            <BlackLink
              title='DOWNLOAD MANIFESTO'
              link='https://oneoone-resource.s3.ap-northeast-2.amazonaws.com/yzed/YZED_MANIFESTO.pdf'
            />
          </div>
        </div>
        <div className='flex flex-col justify-end p-0 lg:pt-16'>
          {model.imageSlug && (
            <ProgressiveImage image={photo.max} preview={photo.mini} alt={photo.id} />
          )}
        </div>
      </div>
      <style jsx>{`
        h1 {
          z-index: 1000;
        }
        img {
          max-height: 444px;
          width: auto;
          max-width: 90%;
        }
      `}</style>
      <style jsx global>{`
        .black-link {
          margin-top: 50px;
        }
        @media (max-width: 768px) {
          .black-link {
            display: block;
            margin-top: 0px;
            margin: 10px auto;
          }
        }
      `}</style>
    </>
  );
};

export default LandingIntroduction;

LandingIntroduction.propTypes = {
  model: PropTypes.object.isRequired,
};
