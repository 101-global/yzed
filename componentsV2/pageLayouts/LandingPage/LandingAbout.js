import React, { useContext, useEffect, useState } from 'react';

import BlackLink from '../../reusableStyledComponents/BlackLink';
import ProgressiveImage from '../../ProgressiveImage';
import YZEDAnimatedLogo from '../../reusableStyledComponents/YZEDAnimatedSVG';
import image1 from '../../../public/HQ_IMGS2_0120.png';
import image2 from '../../../public/HQ_IMAGES8_0032.png';
import selfExpressionWhite from '../../../public/self-expression.svg';
import theme from '../../../utilities/theme';

const LandingAbout = () => {
  return (
    <>
      <div className='main-about-container pb-32 align-center justify-center w-full'>
        <div className='text-container grid grid-cols-1 lg:block align-center justify-center w-full lg:w-4/5 mx-auto my-0 relative'>
          <h1 className='text-3xl lg:text-4xl px-4 lg:pb-8 lg:p-0 lg:text-4xl w-full lg:w-5/12 inline-block text-center lg:text-left'>
            “The desire to make fashion more accessible to people.”
          </h1>
          <p className='text-xl w-full lg:w-6/12 block lg:inline-block relative lg:absolute right-o font-hairline z-50 right-0 px-4 py-12 lg:pb-8 lg:pl-8 lg:pt-0 lg:pr-0 right-0 text-justify'>
            We are an international team of four, based in Seoul, South Korea. We became united by
            the desire to make fashion more accessible to people, which has continued to be our
            team’s top priority. We strongly believe that a change of status quo is imminent in the
            fashion industry - and that it’s going to be catal<strong>yzed</strong> by none other
            than digital garments created in 3D & AR. We strive to create a community where our
            users self-expression leads our decisions.
          </p>
        </div>
        <div className='bottom-section relative'>
          <div className='image-container w-full'>
            <div className='small-image-container relative lg:absolute right-0  lg:w-2/5 mb-8 lg:mb-0'>
              <ProgressiveImage image={image2} />
            </div>
            <div className='large-image-container  lg:w-9/12'>
              <ProgressiveImage image={image1} />
            </div>
            <div className='word-mark'>
              <img src={selfExpressionWhite} alt='contact us' />
            </div>
          </div>
        </div>
      </div>

      <style jsx>{`
        .main-about-container {
          color: white;
        }
        h1,
        p {
          color: ${theme.colors.white};
          background: ${theme.colors.black};
        }
        .text-container {
          padding-top: 100px;
        }
        h1 {
          font-weight: 600;
        }
        p {
          font-weight: 100;
        }
        .small-image-container {
          top: calc(50% - 200px);
        }
        .word-mark img {
          overflow: hidden;
          position: absolute;
          bottom: -30px;
          right: 80px;
          mix-blend-mode: exclusion;
          filter: contrast(175%) grayscale(100%);
        }
      `}</style>
      <style jsx global>{``}</style>
    </>
  );
};

export default LandingAbout;
