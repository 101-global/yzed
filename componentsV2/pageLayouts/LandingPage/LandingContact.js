import React, { useContext, useEffect, useState } from 'react';

import ContactAnimated from '../../reusableStyledComponents/ContactAnimatedSVG';
import contactText from '../../../public/contact_us.svg';
import theme from '../../../utilities/theme';

const LandingContact = () => {
  return (
    <>
      <>
        <div className='main-contact-container px-dsk grid grid-cols-1 lg:grid-cols-2 gap-8 justify-center align-center'>
          <div className='title-section flex flex-col justify-center align-center'>
            <h1 className='text-3xl lg:text-5xl'>Let's Work Together!</h1>
            <ContactAnimated />
          </div>
          <div className='content-section flex flex-col justify-center align-center'>
            <p className='text-xl'>
              Dive into a mixed-reality world with a team of professionals in 3D & graphic design,
              UI/UX, trend forecasting, as well as front and back-end programming. YZED is turning
              your exquisite physical products into digital-dazzle, leading your brand into the
              digital era.
            </p>
            <div className='signature'>
              <p className='text-xl'>Interested in working with us? </p>
              <p className='text-xl'>
                {' '}
                Please contact us at <a className='email-link'>yzed@101.global</a>
              </p>
            </div>
          </div>
        </div>
      </>
      <style jsx>{`
        .main-contact-container {
          width: 100%;
        }
        .title-section,
        .content-section {
          position: relative;
        }
        .content-section p {
          font-weight: 200;
        }
        .signature {
          margin-top: 30px;
        }
        a {
          font-weight: 600;
        }
        h1 {
          text-align: right;

          font-weight: 500;
          z-index: 500;
          margin-right: 20px;
        }

        a.email-link {
          display: inline-block;
          cursor: pointer;
        }
        a.email-link:after {
          content: '';
          background-image: linear-gradient(to right, #a891d9, #73cfdb, #a891d9, #73cfdb);
          background-size: 260% 200%;
          width: 110%;
          margin-left: -5%;
          margin-top: -10px;
          height: 12px;
          display: block;
        }
        a.email-link:hover:after {
          animation: gradient 2s ease infinite;
          animation-direction: alternate;
        }
        @keyframes gradient {
          0% {
            background-position: 0% 50%;
          }
          50% {
            background-position: 100% 50%;
          }
          100% {
            background-position: 0% 50%;
          }
        }
      `}</style>
    </>
  );
};

export default LandingContact;
