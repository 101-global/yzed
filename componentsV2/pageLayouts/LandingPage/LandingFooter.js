import React, { useContext, useEffect, useState } from 'react';

import theme from '../../../utilities/theme';

const LandingFooter = () => {
  return (
    <>
      <footer className='text-left'>
        <p className='p-2 lg:px-dsk'>
          <strong>YZED</strong> - All rights reserved to{' '}
          <a href='https://www.101.global/?lang=en' target='_blank' rel='noopener noreferrer'>
            101Global
          </a>{' '}
          and{' '}
          <a href='https://www.hanwha.com/en.html' target='_blank' rel='noopener noreferrer'>
            Hanwha
          </a>
          .
        </p>
      </footer>
      <style jsx>{`
        footer p {
          font-weight: 100;
          font-size: 1.2rem;
          background: ${theme.colors.black};
          color: ${theme.colors.white};

          width: 100%;
          letter-spacing: 0.02em;
        }

        footer p a {
          margin: 0 5px;
          font-weight: 600;
        }
        footer p strong {
          display: inline-block;
        }
        footer p strong:after {
          content: '';
          background-image: linear-gradient(to right, #a891d9, #73cfdb);
          width: 110%;
          margin-left: -5%;
          margin-top: -12px;
          height: 12px;
          display: block;
        }
      `}</style>
    </>
  );
};

export default LandingFooter;
