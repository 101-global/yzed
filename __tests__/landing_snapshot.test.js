jest.mock('../components/reusableStyledComponents/YZEDAnimatedSVG.js', () => () =>
  'YZED Animated SVG Component'
);
jest.mock('../components/reusableStyledComponents/ContactAnimatedSVG.js', () => () =>
  'YZED Animated SVG Component'
);

import LandingAbout from '../components/pageLayouts/LandingPage/LandingAbout';
import LandingContact from '../components/pageLayouts/LandingPage/LandingContact';
import LandingFooter from '../components/pageLayouts/LandingPage/LandingFooter';
import LandingIntroduction from '../components/pageLayouts/LandingPage/LandingIntroduction';
import React from 'react';
import { model } from '../__mocks__/dataMock';
import renderer from 'react-test-renderer';

it('renders introduction unchanged', () => {
  const wrapper = renderer.create(<LandingIntroduction model={model} />).toJSON();
  expect(wrapper).toMatchSnapshot();
});

it('renders landing about section unchanged', () => {
  const tree = renderer.create(<LandingAbout model={model} />).toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders contact unchanged', () => {
  const tree = renderer.create(<LandingContact />).toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders footer section unchanged', () => {
  const tree = renderer.create(<LandingFooter />).toJSON();
  expect(tree).toMatchSnapshot();
});
