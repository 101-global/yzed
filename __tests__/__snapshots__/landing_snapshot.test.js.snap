// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders contact unchanged 1`] = `
<div
  className="jsx-4172434120 px-4 lg:px-dsk flex flex-col lg:flex-row justify-center lg:justify-between align-center w-full"
>
  <div
    className="jsx-4172434120 center-flex-col lg:w-6/12"
  >
    <h1
      className="jsx-4172434120 text-3xl lg:text-5xl m-0 lg:mr-16 text-center lg:text-right mt-20 mb-20 lg:my-0 font-semibold z-fiveHund"
    >
      Let's Work Together!
    </h1>
    YZED Animated SVG Component
  </div>
  <div
    className="jsx-4172434120 center-flex-col lg:w-6/12 md:mb-16 lg:mb-0 text-justify lg:text-left"
  >
    <p
      className="jsx-4172434120 para-standard"
    >
      Dive into a mixed-reality world with a team of professionals in 3D & graphic design, UI/UX, trend forecasting, as well as front and back-end programming. YZED is turning your exquisite physical products into digital-dazzle, leading your brand into the digital era.
    </p>
    <div
      className="jsx-4172434120 signature mt-16 text-left"
    >
      <p
        className="jsx-4172434120 para-standard"
      >
        Interested in working with us? 
      </p>
      <p
        className="jsx-4172434120 para-standard"
      >
        Please contact us at
         
        <a
          className="jsx-4172434120 email-link font-semibold inline-block cursor-pointer"
        >
          yzed@101.global
        </a>
      </p>
    </div>
  </div>
</div>
`;

exports[`renders footer section unchanged 1`] = `
<footer
  className="text-left"
>
  <p
    className="px-2 py-4 lg:px-dsk lg:py-2 lg:text-base text-center lg:text-left align-middle tracking-wide w-full bg-black text-white font-light"
  >
    <a
      aria-label="YZED home"
      className="color-link block lg:inline-block pb-6 pt-2 lg:pb-0 lg:pt-0 align-middle my-0 mx-auto h-3 -mt-1"
      href="/"
    >
      <img
        alt="YZED logo"
        className="yzed-logo align-middle my-0 mx-auto h-3"
        src="File Sub Mock"
      />
    </a>
     
    <span
      className="hidden lg:inline"
    >
      - 
    </span>
    All rights reserved to
     
    <a
      className="font-semibold"
      href="https://www.101.global/?lang=en"
      rel="noopener noreferrer"
      target="_blank"
    >
      101Global
    </a>
     
    and
     
    <a
      className="font-semibold"
      href="https://www.hanwha.com/en.html"
      rel="noopener noreferrer"
      target="_blank"
    >
      Hanwha
    </a>
    .
  </p>
</footer>
`;

exports[`renders introduction unchanged 1`] = `
<div
  className="jsx-1337300742 main-intro-container p-0 lg:pl-dsk grid gap-8 lg:gap-16 sm:grid-cols-1 lg:grid-cols-2 mb-16 lg:mb-0"
>
  <div
    className="jsx-1337300742 flex w-full mt-16 md:mt-24 lg:mt-0 text-center lg:text-left relative"
  >
    <div
      className="jsx-1337300742 flex flex-col justify-between font-black"
    >
      YZED Animated SVG Component
      <h1
        className="jsx-1337300742 pt-16 px-4 lg:px-0 text-4xl lg:text-5xl my-0 mx-auto text-center lg:text-left font-semibold"
      >
        Where Democratic Fashion catalyzes self-discovery.
      </h1>
      <p
        className="jsx-1337300742 para-standard text-justify lg:text-left px-4 lg:px-0 py-12 lg:py-0 my-0 mx-auto lg:mt-24 lg:m-0"
      >
        Democratic Fashion is a practice that minimizes the gap between a designers’ work and the mass market, while considering fair economic and social trade off. YZED is built upon the five most important factors of this practice, which you can read more about in our manifesto.
      </p>
      <a
        className="jsx-4218770882 black-link block lg:inline m-auto lg:m-0 py-2 bg-black text-white border border-black text-xl text-center rounded-full no-underline tracking-wider font-medium hover:text-black hover:bg-white"
        href="https://oneoone-resource.s3.ap-northeast-2.amazonaws.com/yzed/YZED_MANIFESTO.pdf"
        rel="noopener noreferrer"
        target="_blank"
      >
        DOWNLOAD MANIFESTO
      </a>
    </div>
  </div>
  <div
    className="jsx-1337300742 flex flex-col justify-end p-0 lg:pt-16"
  >
    <img
      alt="YZED Detail View 1"
      loading="lazy"
      src="https://oneoone-resource.s3.ap-northeast-2.amazonaws.com/yzed/yzed_image_1_mini-compressor.jpg"
      style={
        Object {
          "background": "grey",
          "filter": "blur(30px)",
          "height": "auto",
          "maxWidth": "100%",
          "overflow": "hidden",
          "transition": "1s filter linear",
          "width": "3000px",
        }
      }
    />
  </div>
</div>
`;

exports[`renders landing about section unchanged 1`] = `
<div
  className="jsx-2873791416 main-about-container py-16 lg:pb-32 lg:pt-0 align-center justify-center w-full"
>
  <div
    className="jsx-2873791416 text-container grid grid-cols-1 lg:block align-center justify-center w-full lg:w-4/5 mx-auto my-0 relative p-0 lg:pt-32"
  >
    <h1
      className="jsx-2873791416 text-3xl lg:text-4xl px-4 lg:pb-8 lg:p-0 lg:text-4xl w-full lg:w-5/12 inline-block text-center lg:text-left font-semibold bg-black text-white"
    >
      “The desire to make fashion more accessible to people.”
    </h1>
    <p
      className="jsx-2873791416 text-xl w-full lg:w-6/12 block lg:inline-block relative lg:absolute right-o font-hairline z-50 right-0 px-4 py-12 lg:pb-8 lg:pl-8 lg:pt-0 lg:pr-0 right-0 text-justify font-light bg-black text-white"
    >
      We are an international team of four, based in Seoul, South Korea. We became united by the desire to make fashion more accessible to people, which has continued to be our team’s top priority. We strongly believe that a change of status quo is imminent in the fashion industry - and that it’s going to be catal
      <strong
        className="jsx-2873791416 font-bold"
      >
        yzed
      </strong>
       by none other than digital garments created in 3D & AR. We strive to create a community where our users self-expression leads our decisions.
    </p>
  </div>
  <div
    className="jsx-2873791416 bottom-section relative"
  >
    <div
      className="jsx-2873791416 image-container w-full"
    >
      <div
        className="jsx-2873791416 small-image-container relative lg:absolute right-0 lg:w-2/5 mb-8 lg:mb-0"
      >
        <img
          alt="YZED Detail View 3"
          loading="lazy"
          src="https://oneoone-resource.s3.ap-northeast-2.amazonaws.com/yzed/yzed_image_3_mini-compressor.jpg"
          style={
            Object {
              "background": "grey",
              "filter": "blur(30px)",
              "height": "auto",
              "maxWidth": "100%",
              "overflow": "hidden",
              "transition": "1s filter linear",
              "width": "3000px",
            }
          }
        />
      </div>
      <div
        className="jsx-2873791416 large-image-container lg:w-9/12"
      >
        <img
          alt="YZED Detail View 2"
          loading="lazy"
          src="https://oneoone-resource.s3.ap-northeast-2.amazonaws.com/yzed/yzed_image_2_mini-compressor.jpg"
          style={
            Object {
              "background": "grey",
              "filter": "blur(30px)",
              "height": "auto",
              "maxWidth": "100%",
              "overflow": "hidden",
              "transition": "1s filter linear",
              "width": "3000px",
            }
          }
        />
      </div>
      <div
        className="jsx-2873791416 word-mark"
      >
        <img
          alt="contact us"
          className="jsx-2873791416"
          src="File Sub Mock"
        />
      </div>
    </div>
  </div>
</div>
`;
